docs/en/observability/cloud-monitoring/azure/monitor-azure-openai.asciidoc

```diff
@@ -0,0 +1,337 @@
+[[monitor-azure-openai]]
+== Monitor Microsoft Azure OpenAI
+
+****
+**New to Elastic?** Follow the steps in our {estc-welcome}/getting-started-observability.html[getting started guide] instead
+of the steps described here. Return to this tutorial after you've learned the basics.
+****
+
+This tutorial shows you how to use the Elastic Azure OpenAI integration, the Azure portal, and {agent} to collect and monitor Azure OpenAI logs and metrics with Elastic {observability}.
+
+[discrete]
+[[azure-openai-what-you-learn]]
+=== What you'll learn
+
+You'll learn how to:
+
+* Set up your Azure instance to allow the Azure OpenAI integration to collect logs and metrics.
+* Configure the Azure OpenAI integration to collect logs and metrics.
+* Install {agent} on your host.
+* View your logs and metrics in {kib} using built-in dashboards and Discover.
+
+[discrete]
+[[azure-openai-set-up-logs]]
+=== Step 1: Set up Azure to collect logs
+
+The Elastic Azure OpenAI integration captures audit logs and request and response logs.
+
+* Audit logs provide a range of information related to the use and management of Azure OpenAI services.
+* Request and response logs provide information about each request made to the service and the corresponding response provided by the service.
+
+For more on the fields ingested from audit and request and response logs, refer to the {integrations-docs}/azure_openai#settings[Azure OpenAI integration] documentation.
+
+Before {agent} can collect your logs and send them to {kib}, complete the following steps in the https://portal.azure.com/[Azure portal]:
+
+. Create an event hub to receive logs exported from the Azure service and make them available to the {agent}.
+. Configure diagnostic settings to send your logs to the event hub.
+. Create a storage account container where the {agent} can store consumer group information.
+
+[discrete]
+[[azure-openai-event-hub]]
+==== Create an event hub
+
+https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-about[Azure Event Hubs]
+is a data streaming platform and event ingestion service that you use to store
+in-flight Azure logs before sending them to {es}. For this tutorial, you create
+a single event hub because you are collecting logs from one service.
+
+To create an Azure event hub:
+
+. Go to the https://portal.azure.com/[Azure portal].
+. Search for and select **Event Hubs**.
+. Click **Create** and create a new Event Hubs namespace. You'll need to create a new resource group, or choose an existing one.
+. Enter the required settings for the namespace and click **Review + create**.
+. Click **Create** to deploy the resource.
+. In the new namespace, click **+ Event Hub** and enter a name for the event hub.
+. Click **Review + create**, and then click **Create** to deploy the resource.
+. Make a note of the namespace and event hub name because you'll use them to configure your integration settings in <<azure-openai-configure-integration>>.
```

Comment:
The event hub namespace is not required to `configure the integration`. It is required in Azure to add new event hub inside the namespace. We can remove the `namespace` from here.

Reply:
Adding reference for why we are not capturing the [namespace](https://docs.elastic.co/integrations/azure#:~:text=You%20should%20use%20the%20event%20hub%20name%20(not%20the%20event%20hub%20namespace%20name)%20as%20a%20value%20for%20the%20eventhub%20option%20in%20the%20integration%20settings.) in the document as configuration parameter.